tgui/packages/tgui/interfaces/CrewCredits/AntagonistsTab.tsx

```diff
@@ -0,0 +1,251 @@
+/**
+ * @file
+ * @copyright 2023
+ * @author Mr. Moriarty (https://github.com/Mister-Moriarty)
+ * @license MIT
+*/
+
+import { Fragment } from 'inferno';
+import { useBackend } from '../../backend';
+import { Box, Collapsible, Divider, Icon, ItemList, LabeledList, Section, Stack } from '../../components';
+import { AntagonistStatisticsProps, AntagonistTabData, VerboseAntagonistProps } from './type';
+
+export const AntagonistsTab = (props, context) => {
+  const { data } = useBackend<AntagonistTabData>(context);
+
+  return (
+    <Box>
+      <Section>
+        <Stack
+          vertical
+          align="center"
+          my={3}>
+          <Stack.Item
+            mb={-2.5}
+            italic>
+            The Game Mode Was:
+          </Stack.Item>
+          <Stack.Item
+            fontSize={2.75}
+            bold>
+            {data.game_mode}
+          </Stack.Item>
+        </Stack>
```

Comment:
Slice this out into its own component, there's a lot of presentation information here that adds unnecessary complexity to reading the parent component to show what's actually just `<GameModeDisplay>{data.game_mode}</GameModeDisplay>`